WrightTools/data/_axis.py

```diff
@@ -90,7 +90,9 @@ def label(self):
         symbol = wt_units.get_symbol(self.units)
         label = r'$\mathsf{' + self.expression
         for v in self.variables:
-            label = label.replace(v.natural_name, '%s_{%s}' % (symbol, v.label))
+            vl = '%s_{%s}' % (symbol, v.label)
+            vl = vl.rstrip('_{}')  # label can be empty, should not propagate trailing underscore
```

Comment:
https://docs.python.org/3.6/library/stdtypes.html?highlight=rstrip#str.rstrip

Reply:
Rstrip will strip any character in the argument string, not just the full sequence